circle.yml

```diff
@@ -36,7 +52,9 @@ dependencies:
       fi &&
       cd ~/osslsigncode && sudo make install
     - make deps build -j $JOBZ
-    - sudo apt-get install cowsay nsis jq
+    - sudo apt-get update
```

Comment:
Assuming these run in order, you might want to do this at the very beginning of this `override` section, as you install other stuff earlier.